### PR TITLE
Task 6 ready for review

### PR DESCRIPTION
**Pull request by KseniaShurina**

### CODE REVIEW

File: HW6/Task6.BL/Program.cs [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+// GetBookByIsbn
+Console.WriteLine();
+
+Console.WriteLine("Get Book By Isbn:");
+var bookByIsbn1 = catalog.GetBookByIsbn("9780307887894"); // book 1
+Console.WriteLine(bookByIsbn1);
+var bookByIsbn2 = catalog.GetBookByIsbn("978-1-59-139619-2"); // book 3
+Console.WriteLine(bookByIsbn2);
+
```
Comment (IgorKheidorov) [inline HW6/Task6.BL/Program.cs:95]: if(input == "XML")
{
repo = new XMLRepo();
}

repo.Save(catalog);
repo.Load();

File: HW6/Task6.BL/Extensions/Entities/BookExtension.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+﻿using Task6.DAL.Entities;
+using Task6.DAL.XMLEntities;
+
```
Comment (IgorKheidorov) [inline HW6/Task6.BL/Extensions/Entities/BookExtension.cs]: Mappers, one class for all extensions

File: HW6/Task6.DAL/Entities/Author.cs [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+{
+    public class Author
+    {
+        public string FirstName { get; }
+        public string LastName { get; }
+        public DateTime? DateOfBirthday { get; }
+
+        public HashSet<Book>? Books { get; } = new();
+
+        public Author(string firstName, string lastName, DateTime? dateOfBirthday, IEnumerable<Book>? books = null)
```
Comment (IgorKheidorov) [inline HW6/Task6.DAL/Entities/Author.cs]: magic numbers

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]
Please refactor

File: HW6/Task6.BL/Program.cs (hunk 1 above)
Comment (KseniaShurina) [reply HW6/Task6.BL/Program.cs:95]: I didn't figure out how to realize it in the code. This is why I created logic with some repos in the Program file and called methods directly

**Review by IgorKheidorov — APPROVED** [review]
Ok)

**Pull request merged**